lib/maze/repeaters/bugsnag_repeater.rb

```diff
@@ -17,10 +17,11 @@ def enabled?
       end
 
       def url_for_request_type
+        traces_endpoint = ENV['MAZE_REPEATER_TRACES_ENDPOINT'] || 'otlp.bugsnag.com/v1/traces'
```

Comment:
will probably want the full endpoint including the protocol, as we are hard coding the protocol to https. but a later task to do anything with traces